docs/INSTALLATION.md

```diff
@@ -1,15 +1,47 @@
 # Installation
 
-<!-- TODO: write document
+## Windows Users
 
-  This document should describe what the student needs to install
-  to allow working on the track on their local system using the CLI.
+Please install [Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/install) and see the next section.
 
-  You can include the installation instructions in this document, but
-  usually it is better to link to a resource with the official installation
-  instructions, to prevent the instructions from becoming outdated.
+## Linux / mac Users
 
-  The contents of this document are displayed on the track's documentation
-  page at `https://exercism.org/docs/tracks/<track>/installation`.
+### Installing with brew
 
-  See https://exercism.org/docs/building/tracks/docs for more information. -->
+This is the easiest way to install Chapel. It will give you a basic working installation. This is enough for Exercism, but it has a few limitations, for example it does not support multi-locale. More advanced users may want to build Chapel from source code (next section).
+
+If you haven't already, install the [brew](https://brew.sh/) package manager.
+
+1. First, make sure brew is up-to-date by running `brew update`
+2. Install Chapel by running `brew install chapel`
+3. Verify the installation was successful by running `chpl --version`
+
+This will install the chapel compiler (`chpl`), the package manager `mason`, that we will use to run the tests, and the documentation generation tool `chpldoc`, this is not needed for exercism, but it is good to have if (when :) ) you want to develop your own libraries in Chapel.
+
+### Building Chapel from source code
+
+More advanced users may want to build Chapel from the source code themselves. To do so,
+
+1. Install the [prerequisites](https://chapel-lang.org/docs/usingchapel/prereqs.html)
+2. [Download](https://chapel-lang.org/download.html) the archive of the latest Chapel release and extract it.
+3. You can next install Chapel into a specific location by doing
```

Comment:
> where is the configure script?
Yes it's already shipped with the archive. I guess you are asking because often one needs to generate the script with `autoconf` or similar? Maybe I can clarify this.

> If I run configure without passing any arguments where will it be installed?
by default, it's installed in `/usr/local/`, I will clarify this!